content/articles/wrap-up-2020.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "2020 CompilED Posts Wrap-up"
+date: 2021-01-18
+type: "post"
+authors: ["ctldev"]
+tags: ["wrap-up","reviews"]
+lede: "Hindsight is 2020, and in this wrap-up, we look back and what we’ve
+learned and written in this extraordinary year."
+shortlede: "Hindsight is 2020, and in this wrap-up, we look back and what we’ve
+learned and written in this extraordinary year."
+poster: "poster-wrap-up-2020.jpg"
+thumbnail: "thumbnail-wrap-up-2020.jpg"
+socmediaimg: "socmediaimg-wrap-up-2020.jpg"
+hiliteimg: "hiliteimg-wrap-up-2020.jpg"
+poster_sourceurl: "https://unsplash.com/photos/n2M2SBQE6Iw"
+poster_source: "Chris Barbalis on Unsplash"
+bookendanimal: "laptop-house"
+---
+
+The year that was, with visually appealing, even pattern of 2-0-2-0, can be
```

Comment:
need an article here for visually, maybe "with a visually appealing" or "with the visually appealing"